packages/tools/devtools/devtools-view/src/test/AudienceStateTable.test.tsx

```diff
@@ -0,0 +1,41 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// Normal usage pattern for @testing-library/jest-dom
+// eslint-disable-next-line import/no-unassigned-import
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import { userEvent } from "@testing-library/user-event";
+import React from "react";
+
+import { AudienceStateTable } from "../components/index.js";
+
+import { assertNoAccessibilityViolations } from "./utils/index.js";
+
+describe("AudienceStateTable Accessibility Check", () => {
+	it("AudienceStateTable is accessible", async () => {
+		const { container } = render(<AudienceStateTable audienceStateItems={[]} />);
+		await assertNoAccessibilityViolations(container);
+	});
+
+	it("Can tab/arrow navigate through the AudienceStateTable", async () => {
+		render(<AudienceStateTable audienceStateItems={[]} />);
+
+		const user = userEvent.setup();
+		// Focus on the first interactive element (dropdown theme selector)
```

Comment:
I think this comment applies to the `SettingsView.test.tsx`, so we should probably remove it.